pkg/api/validation/validation_test.go

```diff
@@ -3045,7 +3045,6 @@ func TestValidateVolumeMounts(t *testing.T) {
 		{Name: "abc-123", MountPath: "/bab", SubPath: "baz"},
 		{Name: "abc-123", MountPath: "/bac", SubPath: ".baz"},
 		{Name: "abc-123", MountPath: "/bad", SubPath: "..baz"},
-		{Name: "abc", MountPath: "c:/foo/bar"},
```

Comment:
@sttts It's true. Apiserver can be run on various OS/Arch. Currently all the CIs are running on Linux. Such windows-like absolute path "c:/foo/bar" is considered as relative path on Linux.